pithos/PreferencesPithosDialog.py

```diff
@@ -213,11 +210,10 @@ def cb(success):
 
             email = self.email_entry.get_text()
             password = self.password_entry.get_text()
-            pandora_one = self.pandora_one_checkbutton.get_active()
 
-            if self.last_email != email or self.last_password != password or self.last_pandora_one != pandora_one:
+            if self.last_email != email or self.last_password != password:
                 SecretService.set_account_password(self.last_email, email, password, cb)
             else:
-                self.settings.revert()
+                self.settings.apply()
```

Comment:
That's a fix from the last bit. Otherwise just changing the audio quality setting has no effect.